inventory/environ.py

```diff
@@ -340,19 +340,22 @@ def getSplunkApps(vars_scope):
     """
     Determine the set of Splunk apps to install as union of defaults.yml and environment variables
     """
-    appSet = set()
+    appList = []
```

Comment:
Why an array?  I see you're removing duplicates later in this method.  Is it to preserve the original order?

Reply:
Yeah, per the issue, it seems like certain apps can impede REST API functionality when installed in a randomized order. I didn't look into the specifics of it - my guess is this might be things that intercept REST handlers or add proxying or change auth backends? 

In any case, I don't why we can't expose an ordered installation if that's what folks want.